contributed_definitions/nyaml/NXdata_mpes_detector.yaml

```diff
@@ -0,0 +1,157 @@
+category: base
+doc: |
+  :ref:`NXdata_mpes_detector` describes the plottable data and related dimension scales 
+  for raw detector data in MPES experiments.
+  
+  It extends the NXdata class and provides a glossary of explicitly named axis names
+  which are typical for raw MPES data.
+type: group
+NXdata_mpes_detector(NXdata):
```

Comment:
Not really, the fields are similar, but the docstrings differ oftentimes (because we distinguish calibrated data in NXdata_mpes and uncalibrated data in NXdata_mpes_detector). Maybe in the future we can think about changing this.

Reply:
Fine with that maybe then wrap NXdata_mpes(_detector) individual in an NXprocess, in APM there are now several examples for this, each step of the processing pipeline is a specialized NXprocess which has its own (atom-algorithm-step-specific) NXdata instance that is though and should be well-defined